README.md

```diff
@@ -201,6 +201,7 @@ For example, if a locally running service provided that feature, the following c
 Bumblebee hosted chat models are supported. There is built-in support for Llama 2, Mistral, and Zephyr models.
 
 Currently, function calling is NOT supported with these models.
```

Comment:
Maybe rephrase this to 
"Currently, function calling is only supported for llama 3.1  Tool calling for Llama 2, Mistral, and Zephyr is NOT supported.